go.mod

```diff
@@ -262,6 +262,7 @@ replace (
 	github.com/apoydence/eachers => github.com/poy/eachers v0.0.0-20181020210610-23942921fe77 //indirect, see https://github.com/elastic/beats/pull/29780 for details.
 	github.com/dop251/goja => github.com/andrewkroh/goja v0.0.0-20190128172624-dd2ac4456e20
 	github.com/dop251/goja_nodejs => github.com/dop251/goja_nodejs v0.0.0-20171011081505-adff31b136e6
+	github.com/elastic/csp-security-policies => github.com/uri-weisman/csp-security-policies v0.0.18-go-lib.0.20220801134428-747e21fc1617
```

Comment:
will be replaced after https://github.com/elastic/security-team/issues/4452 is merged.

Reply:
You mean this one? https://github.com/elastic/csp-security-policies/pull/116/files